addons/hr_timesheet/static/src/js/timesheet_graph_view.js

```diff
@@ -0,0 +1,22 @@
+odoo.define('hr_timesheet.timesheet_graph_view', function (require) {
+    "use strict";
+
+    const GraphView = require('web.GraphView');
+    var viewRegistry = require('web.view_registry');
+    
+    const TimesheetGraphView = GraphView.extend({
+        /**
+         * @override
+         */
+        init: function (viewInfo, params) {
+            this._super.apply(this, arguments);
+
+            this.loadParams.fields["unit_amount"].string = this.controllerParams.measures[0].description;
+            this.loadParams.fields["amount"].string = this.controllerParams.measures[1].description;
```

Comment:
Do we always have (at least) 2 entries in measures? You might need to check if [0] and [1] are not undefined to prevent any traceback :-)